### PR TITLE
RUM-1828 feat! (128 bit trace id) update changelog

### PR DESCRIPTION
**Pull request by ganeshnj**
### What and why?

Changelog entry was missed.

### How?

Added changelog.

### Review checklist
- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [ ] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing changes

### Custom CI job configuration (optional)
- [ ] Run unit tests for Core, RUM, Trace, Logs, CR and WVT
- [ ] Run unit tests for Session Replay
- [ ] Run integration tests
- [ ] Run smoke tests
- [ ] Run tests for `tools/`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 * [FEATURE] Call RUM's `errorEventMapper` for crashes. See [#1742][]
 * [FEATURE] Support calling log event mapper for crashes. See [#1741][]
 * [FIX] Fix crash in `NetworkInstrumentationFeature`. See [#1767][]
+* [FEATURE] Add support for 128 bit trace IDs. See [#1721][]
+
+[#1759]:
```
Comment (ncreated) [inline CHANGELOG.md:9]: blocking/ There's something wrong with this one

**Review by ncreated — CHANGES_REQUESTED** [review]
Needs a fix 🙂

File: CHANGELOG.md (hunk 1 above)
Comment (ganeshnj) [reply CHANGELOG.md]: there is a reason one must review first their own PRs :)

**Review by maxep — COMMENTED** [review]
Can we hold on this one until the [release](https://github.com/DataDog/dd-sdk-ios/pull/1770) gets merged to develop? 🙏

**Comment by ganeshnj** [comment]
> Can we hold on this one until the [release](https://github.com/DataDog/dd-sdk-ios/pull/1770) gets merged to develop? 🙏

not sure this would help as the main PR is already merged https://github.com/DataDog/dd-sdk-ios/pull/1721

anyway I can wait.

**Review by ncreated — DISMISSED** [review]

**Review by ncreated — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 * [FIX] Fix crash in `NetworkInstrumentationFeature`. See [#1767][]
 * [FIX] Remove modulemap. See [#1746][]
 * [FIX] Expose objc interfaces in Session Replay module. See [#1697][]
+* [FEATURE] Add support for 128 bit trace IDs. See [#1721][]
```
Comment (maxep) [inline CHANGELOG.md:12]: It should be in the `Unreleased` section now.

**Review by maxep — APPROVED** [review]
Thank you :)

**Review by ncreated — APPROVED** [review]

**Pull request merged**